README.md

```diff
@@ -445,6 +445,40 @@ bouncing network interfaces before they are active. This can be done by setting
           ports: [eth1, eth2]
 ```
 
+18) Routes and rules can be a mix of strings and hashes (a.k.a. dict, mapping, associative array, etc.) allowing
+    the role to handle rules and routes outside the expected formats.
+
+    The example below configures an interface, to act as a second interface so that any traffic that hits it is returned via the same interface. In addition internal traffic is marked so it can be picked up by some other service (like HAproxy).
+
+### host_vars/proxy1
+```yaml
+interfaces_route_tables:
+    - name: frontend
+      id: 200
```

Comment:
Should it be 100?

Reply:
oops, no there should be a 
```
    - name: marktable
      id: 100
```

I mixed two 'sources' and didn't standardise on named tables